### ISSUE TITLE
[request] Release new build?

### ISSUE THREAD

**Issue by L-a-n-g-o-l-i-e-r-s** [Bioruebe/UniExtract2#45]
Hi there,

Would you consider compiling a new build for release with the fixes you have done?

Please and thank you!

**Comment by vatterspun** [comment]
Are you just asking if the author is still working on this program?

**Comment by L-a-n-g-o-l-i-e-r-s** [comment]
No, I'm asking them to make a new build based on the commits that have been made since. I haven't checked to see if they have released a new build recently. 

An example I think is they fixed the 7zip password bug, but the fixes haven't been released in a new build. :( 

2.0.0 beta 4 was released in October 2016. :((( Lots of commits since.

Thanks for the hard work author/collaborators.

**Comment by muellerlukas** [comment]
It's written in AutoIt. So it's really, really simple to compile it by yourself.

**Comment by L-a-n-g-o-l-i-e-r-s** [comment]
That's good to know, however there are people whom don't know how to do that, if it is as simple as you say, it should be little to no trouble to upload a new build under the release section?

**Comment by mirh** [comment]
I was going to tell you that if you never try you'll never learn. 
And that dev has probably better things to spend his time over. 
And that, besides, it's easy as pieing to use Aut2exe and make UniExtract exe. 

...

Though, I seriously don't know if I'm supposed to download all the external dependencies/binaries by hand.

**Comment by JonnyThree** [comment]
hello i'm using this tool a lot, for exploring the content of nsis, inno and msi installers before the decision of making they part of my os (sorry for my bad english). i find that some binary files inside are a bit outdated, and sometimes extracting msi is impossible, because i can't choose the right way to do this.
i discovered another "branch" of uniextract, tagged as "Koros" version, is this project connected with him in some way?
This is the source of my question:
https://www.portablefreeware.com/forums/viewtopic.php?p=78443

**Comment by L-a-n-g-o-l-i-e-r-s** [comment]
@JonnyThree No this is a separate project or "mod". Different author. 

@mirh Yes, dependencies need to be downloaded as I asked my friend to compile it and it only produced the main installer/executable. (I just dropped the new version in the old folder, but honestly, I imagine the plugins need updating.)

**Comment by Bioruebe** [comment]
> if it is as simple as you say, it should be little to no trouble to upload a new build

I recently addressed a really far-reaching issue: on some system configurations UniExtract could not find any of the extractors in the bin directory. As a result I had to change each extractor call in the source code and - even worse - test all extractions. This is why there has been no release in the last months. I don't want to upload broken versions...

> I just dropped the new version in the old folder

That's the correct way :) Just download the latest release and overwrite the main exe with the newly compiled one. Plugins need to be updated the same way (manually), but most of the time this isn't neccessary. It's very rare that they stop working.
